src/app/dashboard/dashboard.component.ts

```diff
@@ -6,6 +6,7 @@ import { UserService } from '../shared/user.service';
 @Component({
   template: `
     <div id="greeting">Hi, {{name}}</div>
+    <div style="float:right"><a href="manager" i18n class="btn btn-primary" >Manager View</a></div>

```

Comment:
Let's wait until we have better direction from the designers before styling.

Also please don't use inline styles unless it's absolutely necessary.